conf/scylla.yaml

```diff
@@ -590,3 +590,10 @@ consistent_cluster_management: true
 # New clusters have this option set to `true` by scylla.yaml (which overrides the default `warn`)
 # to make sure that trying to create an invalid view causes an error.
 strict_is_not_null_in_views: true
+
+# If set to true, configuration parameters defined with LiveUpdate option can be updated in the runtime.
+# If we don't want any configuration parameter to be changed in the runtime by any means, i.e.
+# reloading scylla.yaml with SIGHUP, updating system.config virtual table or calling API,
+# this option should be set to false. This option should be set to false for e.g. cloud users, for whom
+# scylla's configuration should be changed only by support engineers.
+live_updates_enabled: true
```

Comment:
@tzach should we, by default, allow users to update configuration parameters in CQL?

Reply:
@tzach ping

Reply:
@tzach agreed with my comment to let on-premise (oss & enterprise) users change config params via CQL, so we can leave this param set to `true` by default, and it'll have to be set to `false` for cloud deployments.